index.html

```diff
@@ -44,8 +44,138 @@ <h6><a href="">LET'S CONNECT</a></h6>
                 </ul>
             </aside>
         </section>
-       
     </main>
+
+    1
+    <section class = "works">
```

Comment:
As defined by the project requirements, this work section should be wrapped with one main section and it should have a display: grid 🏷️